### PR TITLE
Moving examples to an appendix

### PR DESCRIPTION
**Pull request by castelao**
Keep the document clean and short.

On the top of the example files, we should add plenty of examples in the document to help the reader, such as the ones added by @vturpin. To avoid a way too long document, what about collecting the examples in the appendix?

### CODE REVIEW

**Review by soerenthomsen — CHANGES_REQUESTED** [review]
For me it makes sense to move the examples into the appendix. Although one example might be helpful upfront? 

When working on these examples it makes sense to make them congruent. 
i.e. for example 1 "=" is used and for the next ones " : ". Should be the same for all examples.

See comments directly in the text

File: OG_Format.adoc [hunk 1]
```diff
@@ -911,3 +862,58 @@ _Operational phase:_ 3 months – Oct 2022 to Dec 2023
...
 [[_heading=h.1egqt2p]]Glider missions not delivering OG1.0 will not be considered as part of the OceanGliders program. It will be encouraged that legacy files be converted and added to OceanGliders final repository
+
+[appendix]
+== Examples
+
+=== Platform, Programme, and Network
+
+Example 1:
+
+* platform (i.e. glider mission) = kraken_20210205
```
Comment (soerenthomsen) [inline OG_Format.adoc:873]: here we write " ... = ... "

File: OG_Format.adoc [hunk 2]
```diff
@@ -911,3 +862,58 @@ _Operational phase:_ 3 months – Oct 2022 to Dec 2023
...
+* Programme = MOOSE glider program
+* Sites = MOOSE_T00, MOOSET_02
+* Networks = Mediterranean Ocean Observing Systems for the Environment (MOOSE), Boundary Ocean Observing Network (BOON), Water Transformation task team”
+
+Example 2:
+
+* platform : sdeep09_sdeep04_20200929
+* Programme : SOCIB Glider Programme
```
Comment (soerenthomsen) [inline OG_Format.adoc:881]: Here ". : ." 

If we decide to use ":" each time the space before : should be removed

i.e. everywhere
```suggestion
* Programme: SOCIB Glider Programme
```

**Review by soerenthomsen — CHANGES_REQUESTED** [review]

**Review by justinbuck — CHANGES_REQUESTED** [review]
Does the main body of the document introduce the examples and reference the appendix

**Comment by castelao** [comment]
This PR is related to #26 , which might require some changes on the examples listed here.

**Review by justinbuck — APPROVED** [review]
requested changes added, approving

**Review by soerenthomsen — APPROVED** [review]

**Pull request merged**